tutorials/advanced_features/Simulating Sequences with Errors and Noise.ipynb

```diff
@@ -1,36 +1,49 @@
 {
```

Comment:
<p>"You can see the different<strong> types of</strong> noises <strong>supported for each basis by calling</strong><code><strong>SimConfig.supported_noises</strong></code>	as follows:"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/460/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>